DCOS/utils/wsmancmd.py

```diff
@@ -135,14 +143,19 @@ def get_url(url, host, use_ssl, port):
 def main():
     exit_code = 1
     (args_ok, url, host, use_ssl, port, auth, username, password,
-     cert_pem, cert_key_pem, cmd, is_powershell_cmd) = parse_args()
+     cert_pem, cert_key_pem, cmd, is_powershell_cmd, is_powershell_script) = parse_args()
     if args_ok:
         url = get_url(url, host, use_ssl, port)
         if is_powershell_cmd:
             cmd = ["powershell.exe",
                    "-ExecutionPolicy", "RemoteSigned", "-NonInteractive",
                    "-EncodedCommand",
                    base64.b64encode((" ".join(cmd)).encode("utf-16-le"))]
+        if is_powershell_script:
```

Comment:
The above code works anyway. Of course, if we passed `--file`, then it doesn't join anything, but it won't fail either.

I would recommend that we remove this. Makes the code cleaner to have one less `if`.